crates/storage/src/counter.rs

```diff
@@ -124,16 +126,16 @@ mod tests {
         Uint512::ONE;
         "uint512"
     )]
-    #[test_case(
-        Int128::new_negative(Uint128::ONE),
-        Int128::new_negative(Uint128::ONE);
-        "int128"
-    )]
-    #[test_case(
-        Dec128::from_str("0.5").unwrap(),
-        Dec128::from_str("1.5").unwrap();
-        "dec128"
-    )]
+    // #[test_case(
+    //     Int128::new_negative(Uint128::ONE),
+    //     Int128::new_negative(Uint128::ONE);
+    //     "int128"
+    // )]
+    // #[test_case(
+    //     Dec128::from_str("0.5").unwrap(),
+    //     Dec128::from_str("1.5").unwrap();
+    //     "dec128"
+    // )]
```

Comment:
TODO: These test cases also relies on signed numbers. Uncomment once we have signed numbers.